WcaOnRails/config/initializers/devise.rb

```diff
@@ -7,6 +7,8 @@
   # confirmation, reset password and unlock tokens in the database.
   # config.secret_key = '0d6561d9be4e102bfbed461d523b94b70c4bebe0fdb3fcb60b61437a0f8d60e6988168085dfadcabbb3b3b56ead695621caae23e6be52db01acbc5c6e52775ff'
 
+  config.parent_controller = 'ApplicationController'
```

Comment:
I prefer `ApplicationController.name` to dead strings for class names. 

Less important here, since `ApplicationController` will hardly change its name, but I still think it's substantially better.